### ISSUE TITLE
[Components and pattern standards] Your content or feature doesn't use standard global navigation components or is missing a standard navigation component.  (04.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#64777]
#### VFS team name
Veteran Facing Forms

#### VFS product name
Form 21-0845 Authorization to Disclose Personal Information to Third Party

#### Point of Contact/Reviewers
Erin White - @erinrwhite - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content or feature doesn't use standard global navigation components or is missing a standard navigation component. 

### Issue Details
Breadcrumbs are missing.

### VA.gov Experience Standard
[Category Number 04, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Please add breadcrumbs per the [IA guidance from CAIA for this form](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/form-digitization-21-0845.md).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by tlei123** [comment]
I'll be fixing this via #64773, so as to open 1 single content-build PR for quicker external review/approval turnaround.

**Comment by tlei123** [comment]
PRs opened:

- [vets-website PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25538)
- [content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1704)

Will ping Platform tomorrow (Friday) for their reviews/approvals.

**Comment by tlei123** [comment]
[vets-website PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25538) merged.  [content-build](https://github.com/department-of-veterans-affairs/content-build/pull/1704) PR should be reviewed & approved by Platform by end of today.

https://staging.va.gov/supporting-forms-for-claims/third-party-authorization-form-21-0845 should be working by Monday.

**Comment by tlei123** [comment]
[content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1704) merged.  Updated URL should be active in Staging by Monday, maybe even today by ~4:30 E.

https://staging.va.gov/supporting-forms-for-claims/third-party-authorization-form-21-0845

**Comment by tlei123** [comment]
@PSIEBENLIST , breadcrumbs now up in Staging:
https://staging.va.gov/supporting-forms-for-claims/third-party-authorization-form-21-0845

Once the merged devops PR gets deployed Wed afternoon, we can directly hit https://staging.va.gov/supporting-forms-for-claims/third-party-authorization-form-21-0845/introduction

**Comment by jeana-adhoc** [comment]
I've confirmed that the breadcrumb and URL have been updated
<img width="859" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/fa2a5dbd-df0c-42b8-a45a-a2d29ad30f2e"> and it matches the [IA Spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/form-digitization-21-0845.md#urls-and-breadcrumbs)

**Comment by tlei123** [comment]
Closing this now; validated in Staging.
